setup.py

```diff
@@ -120,7 +120,6 @@ def can_import(module_name):
     "Bio.Align.Applications",
     "Bio.Align.substitution_matrices",
     "Bio.AlignIO",
-    "Bio.Alphabet",
```

Comment:
If you want to keep a ``Bio.Alphabet`` stub, we need to keep it on the list of installed packages.